### ISSUE TITLE
Controls don't work until windows loses/gains focus

### ISSUE THREAD

**Issue by KushtrimPacaj** [Genymobile/scrcpy#1178]
Hi
Whenever I start scrcpy, the controls ( swipe/click ) don't work...
If the scrcpy windows loses focus (e.g. I click outside of it ), and then regains ( e.g. I click inside it ), then they start working.
You can see the behavior in the video here: https://www.youtube.com/watch?v=VS13SsciXMg

Versions:
scrcpy 1.12.1

dependencies:
 - SDL 2.0.10
 - libavcodec 58.54.100
 - libavformat 58.29.100
 - libavutil 56.31.100

MacOS Catalina 10.15.3 (19D76).  MacBook Pro (16-inch, 2019)

**Comment by rom1v** [comment]
Weird, your window is started without focus (the title bar is not highlighted).

Please try this:
```diff
diff --git a/app/src/screen.c b/app/src/screen.c
index beb1075..2a946ca 100644
--- a/app/src/screen.c
+++ b/app/src/screen.c
@@ -236,6 +236,7 @@ screen_init_rendering(struct screen *screen, const char *window_title,
 void
 screen_show_window(struct screen *screen) {
     SDL_ShowWindow(screen->window);
+    SDL_RaiseWindow(screen->window);
 }
 
 void
```

**Comment by KushtrimPacaj** [comment]
Cloned and built with this change, problem is still happening ...

As I was testing lots of times, it seems that the error doesn't always happen even in latest release, just most of the time. Same with this change.

Any logs I can take to help you debug? Where do I turn them on?

**Comment by rom1v** [comment]
It seems to be a window manager issue. Do you have some configuration like "prevent stealing focus" somewhere in your window manager config (in MacOS config)?

In scrcpy, the window is created hidden, and it is shown when the first frame is available. With `SDL_RaiseWindow`, SDL requests (once more) to your window manager to give the focus. :confused:

**Comment by KushtrimPacaj** [comment]
No, I haven't made any such changes ( at least not intentionally ).
Can't find any such option anywhere in System Preferences though, so I can't check.

**Comment by KushtrimPacaj** [comment]
BTW removed that check that makes `screen_show_window` to be called only on the first frame, since I was thinking it might not work sometimes because might  get called sometimes too early before window is shown. 
But even with it called on every frame, sometimes there is no focused gained ( the `SDL_RaiseWindow ` having no effect ).

If you have any other ideas, I'm available to try them :)
